test/util/providers/common.go

```diff
@@ -189,3 +194,59 @@ func IsSnapshotExisted(cloudProvider, cloudCredentialsFile, bslBucket, bslConfig
 	}
 	return nil
 }
+
+func GetVolumeInfoMetadataContent(
+	cloudProvider,
```

Comment:
Should use `objectStoreProvider` as input when getting volumeinfo.json.gz from an object store, `cloudProvider` might not same with `objectStoreProvider` when having test on Kind or other local clusters.

Reply:
Modified.